pymc/distributions/timeseries.py

```diff
@@ -33,6 +36,55 @@
 ]
 
 
+class GaussianRandomWalkRV(RandomVariable):
+    """
+    GaussianRandomWalk Random Variable
+    """
+
+    name = "GaussianRandomWalk"
+    ndim_supp = 0
+    ndims_params = [0, 0, 0, 0]
+    dtype = "floatX"
+    _print_name = ("GaussianRandomWalk", "\\operatorname{GaussianRandomWalk}")
+
+    @classmethod
+    def rng_fn(
+        cls,
+        rng: np.random.RandomState,
+        mu: Union[np.ndarray, float],
+        sigma: Union[np.ndarray, float],
+        init: float,
+        size: int,
```

Comment:
Currently the tests are failing because the size argument provided is not recognized in the perform op. Another size is created somewhere else it seems, leading to the following error

> E       TypeError: rng_fn() takes 6 positional arguments but 7 were given

Here's the closet line where I notice things going wrong.
https://github.com/aesara-devs/aesara/blob/main/aesara/tensor/random/op.py#L363

Any idea of where I'm going wrong? I'm following the guide below as well as the `TruncatedNormal` implementation

https://docs.pymc.io/en/latest/contributing/developer_guide_implementing_distribution.html
https://github.com/pymc-devs/pymc/blob/main/pymc/distributions/continuous.py#L601
![image](https://user-images.githubusercontent.com/7213793/148153891-958e5cbb-a7b6-4a20-bc84-37e53bc80a66.png)